packages/core/types/src/common/config-module.ts

```diff
@@ -48,7 +48,7 @@ export interface AdminOptions {
    * })
    * ```
    */
-  path?: `/${string}`
+  path: `/${string}`
```

Comment:
Won't this result in `path` being required if you want to set any admin options:

```
export default defineConfig({
  admin: {
     path: "/app", <-- This is now required even though I just want to use the default value
   }
})

Reply:
Ohh yeah. Thanks for flagging that.

We basically want to get rid of the defaults we have in multiple places and instead put them within the `defineConfig` and that means the output of `defineConfig` should contain these properties.

But as you mentioned, that makes it required when you define `admin` property. So lemme make it a `DeepPartial` when receiving the input.

Reply:
Should be fixed by this commit. https://github.com/medusajs/medusa/pull/9496/commits/25cd88a087430f8e0754c2dd1209355de7ce724d